### DISCUSSION TITLE
Has anyone configured a graphic tablet on Wayland?

### DISCUSSION THREAD

**Discussion by jhonfromerov** [qtile/qtile#4901] [Q&A] [answered]
Arch wiki [article](https://wiki.archlinux.org/title/Graphics_tablet#Wayland_and_libinput) about it says,  "OpenTabletDriver may also be used if your Wayland environment offers no tablet options". But does Qtile really not have any tablet options?

**Comment by 5ila5** [comment, marked as answer]
As far as I can tell, QTILE currently does not support graphics tablets on Wayland (#4803). I tried to figure out how to add it but the relevant tablet implementations of wlroots seem not to be supported yet by the python bindings provided by pywlroots (https://github.com/flacjacket/pywlroots/issues/191)
